v1/pipeline/common/inputs.yaml

```diff
@@ -0,0 +1,10 @@
+title: Inputs
+type: object
+description: inputs to be used for cd entities
```

Comment:
can we remove the term cd entities from here as we would be using it in other stages as well?
can we also add 
```
propertyNames:
      pattern: "^[a-zA-Z_][0-9a-zA-Z_\\.$-]{0,127}$"
```
at the end of this file parallel to additionalProperties